source/SpinalHDL/Data types/Int.rst

```diff
@@ -4,54 +4,45 @@ UInt/SInt
 =========
 
 Description
-^^^^^^^^^^^
+-----------
 
-The ``UInt``/``SInt`` type corresponds to a vector of bits that can be used for signed/unsigned integer arithmetic.
+The ``UInt``/``SInt`` types are a vectors of bits interpreted as two's complement unsigned/signed integers.
```

Comment:
`are a vectors of bits` maybe `are a vector of bits` or `are vectors of bits` (preferred).